### PR TITLE
Aggregate all GraphQLDocumentErrors and attach source file and position to each of them

### PR DESCRIPTION
**Pull request by DAB0mB**
Related to https://github.com/dotansimha/graphql-code-generator/issues/925

### CODE REVIEW

File: tsconfig.es2015.json [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
-      "noUnusedParameters": true,
-      "resolveJsonModule": true,
-      "importHelpers": true
-    },
-    "include": ["src/**/*.ts"],
-    "exclude": ["node_modules"]
-  }
-  
\ No newline at end of file
+  "compilerOptions": {
```
Comment (DAB0mB) [inline tsconfig.es2015.json]: Here I just fixed the indention

**Comment by dotansimha** [comment]
Looks good. Merging.

File: src/typings.d.ts [hunk 2]
```diff
@@ -1,2 +1,3 @@
+declare module 'aggregate-error';
```
Comment (kamilkisiela) [inline src/typings.d.ts]:
```
"devDependencies": {
  "@types/aggregate-error": "1.0.0"
```

means we don't need it here

Comment (DAB0mB) [reply src/typings.d.ts]: We do, otherwise there's error regards `import`. The type module isn't written well.

Comment (kamilkisiela) [reply src/typings.d.ts]: So then we don't need `@types/aggregate-error`

Comment (DAB0mB) [reply src/typings.d.ts]: `This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.`. When you use this flag, other issues pop up. It's better this way

Comment (kamilkisiela) [reply src/typings.d.ts]: `declare module 'xxx'` makes it `any` so TS doesn't even use `@types/xxx` any more.

Comment (DAB0mB) [reply src/typings.d.ts]: But I don't want it to be `any`, I want it to be `AggregateError`. That's why I use both.

Comment (DAB0mB) [reply src/typings.d.ts]: If you can fix that properly, by all means please do

**Pull request merged**